gpkit/nomials/nomial_math.py

```diff
@@ -473,8 +473,10 @@ def _simplify_posy_ineq(self, exps, cs):
             if not exps[0]:
                 if cs[0] > 1:
                     raise ValueError("infeasible constraint: %s" % self)
-            # for now, we allow tautological monomial constraints (cs[0] <= 1)
-            # because they allow models to impose requirements on variables
+                else:
+                    # allow tautological monomial constraints (cs[0] <= 1)
+                    # because they allow models to impose requirements
+                    return [], []
```

Comment:
I guess so, if they're not expensive to create. I think array not ndarray though -- I for one don't know how to create an empty ndarray:

```python
In [1]: import numpy as np

In [2]: np.ndarray()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-98b15771a5de> in <module>()
----> 1 np.ndarray()

TypeError: Required argument 'shape' (pos 1) not found

In [3]: np.ndarray([])
Out[3]: array(9.223372036854776e+18)

In [4]: np.array([])
Out[4]: array([], dtype=float64)
```